tests/radv/test_radv_ipv6_ra.py

```diff
@@ -24,10 +29,16 @@
 
 """
 
-@pytest.fixture(scope="module")
-def radv_test_setup(duthosts, rand_one_dut_hostname, ptfhost, tbinfo):
-    duthost = duthosts[rand_one_dut_hostname]
+@pytest.fixture(scope="module", autouse=True)
+def radv_test_setup(request, duthosts, ptfhost, tbinfo):
+    if 'dualtor' in tbinfo['topo']['name']:
+        # Select upper ToR to run RADV tests
+        duthost = duthosts[tbinfo['duts'][0]]
+    else:
+        hostname = request.getfixturevalue('rand_one_dut_hostname')
```

Comment:
Suggest moving this usage to the fixture definition (line 33) for consistency since `rand_one_dut_hostname` is not dual ToR specific

Reply:
removed if/else